sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH campaigns AS (
+  SELECT DISTINCT
+    id,
+    name,
+    FORMAT("%s (%s)", name, CAST(id AS string)) AS fenix_compatible_campaign_name
+  FROM
+    `moz-fx-data-bq-fivetran`.google_ads.campaign_history
+),
+install_dou_metrics AS (
+  SELECT
+    fenix_marketing_metrics.adjust_campaign AS fenix_marketing_metrics_adjust_campaign,
+    fenix_marketing_metrics.submission_date AS date,
+    COALESCE(SUM(fenix_marketing_metrics.new_profiles), 0) AS new_profiles_sum,
+    COALESCE(SUM(fenix_marketing_metrics.dau), 0) AS dau_sum
+  FROM
+    `moz-fx-data-shared-prod.fenix.marketing_attributable_metrics` AS fenix_marketing_metrics
+  GROUP BY
+    fenix_marketing_metrics_adjust_campaign,
+    date
+)
+SELECT
+  campaigns.name AS campaign_name,
+  stats.date AS date,
+  -- Total spend per-campaign
+  stats.cost_micros AS campaign_spend_in_microcents,
+  -- Impressions and clicks over time for each campaign (clicks of our ads)
+  impressions AS ad_impressions,
+  install_dou_metrics.new_profiles_sum AS installs,
+  install_dou_metrics.dau_sum AS dous,
+  clicks AS ad_clicks,
+  -- Cost per-install for each campaign ($/new profiles)
+  CASE
+  WHEN
+    install_dou_metrics.new_profiles_sum = 0
+  THEN
+    0
+  ELSE
+    stats.cost_micros / install_dou_metrics.new_profiles_sum
+  END
+  AS cost_per_install_microcents,
```

Comment:
Let's calculate these ratios in the view

Reply:
Ah, you mean as opposed to doing them in here?

I figured we'd convert these to dollars in the view from micro...microdollars, I guess, but I'm happy to add this calculation to the list.

Though I'd be interested to hear, for my edification, your thinking on what ends up in the table versus the view. Are these metrics a thing we might need to change moving forward, and the CM process is easier in Looker than bq-etl or something?

Reply:
Yeah, I suppose my thought process is summed up as: materialize the components, and lay them out clearly in the view. For a rate (e.g. cost-per-install), the numerator is $ and the denominator is installs, so they should be materialized, and the biz logic can have them in the view.

In Looker, we'd need them separately anyways, since it will be `SUM(cost) / SUM(installs)` to get the rate (grouped by whatever the user chooses in the explore).

Reply:
That makes sense to me! I'll remove this column from the table and we'll calculate it in the view.

Reply:
For now, no. We could eventually join with the conversions table to get conversion counts per-campaign, but I'd rather get cost info first.